pkg/oc/oc.go

```diff
@@ -117,3 +118,17 @@ func CreateSecretsForTektonResults(){
 	//creating secret with generated certificate
 	cmd.MustSucceed("oc", "create", "secret", "tls", "-n", "openshift-pipelines", "tekton-results-tls", "--cert=cert.pem", "--key=key.pem")
 }
+
+func EnsureResutsReady(){
+	cmd.MustSucceed("oc", "wait", "--for=condition=Ready", "tektoninstallerset", "-l", "operator.tekton.dev/type=result")
+}
+
+func GetLogsAndAnnotaions(resorceType string) {
```

Comment:
1. We actually don't need to pollute test log with pipeline logs, `tkn p start --showlog` was the easiest solution for waiting for pipeline run end in the original Bash script. You can use step `Verify pipelinerun` here
2. You get annotations but don't do anything with it :)